### DISCUSSION TITLE
Multiplier Value in Ocular Plugin

### DISCUSSION THREAD

**Discussion by ukmartin** [Stellarium/stellarium#1861] [General]
Hi. My main aim is astrophotography, (as a new hobby) and in setting Stellarium up, I have added my camera lenses into the Lenses section of the Ocular Plugin.

What I want to know is what value to put for the Multiplier values. I have a Canon EOS7D Mkii, which the Sensors tab knows about, but I want to add my Canon 17-85mm, Sigma 70-300mm and Sigma 150-600mm lenses into the Lenses tab. Typing the text in wasn't a problem, but what do I put in for the multiplier value?

Thanks in advance for your assistance,

Martin

**Comment by gzotti** [comment]
Your camera lenses have to be configured as "Telescopes" while "Lenses" are meant for focal reducers or extenders (Shapley or Barlow lenses, respectively).
